PubNub/Core/PubNub+Publish.h

```diff
@@ -483,6 +485,9 @@ typedef void(^PNMessageSizeCalculationCompletionBlock)(NSInteger size);
   mobilePushPayload:(NSDictionary *)payloads storeInHistory:(BOOL)shouldStore
          compressed:(BOOL)compressed withCompletion:(PNPublishCompletionBlock)block;
 
+- (void)    publish:(id)message toChannel:(NSString *)channel
+  mobilePushPayload:(NSDictionary *)payloads storeInHistory:(BOOL)shouldStore
+         compressed:(BOOL)compressed withMetadata:(NSDictionary *)metadata withCompletion:(PNPublishCompletionBlock)block;
```

Comment:
@jzucker2 I've noticed what meta has been added only to two methods, but there is a lot more in public header.